protobuf/plugin/deepcopy/test/deepcopypb_test.go

```diff
@@ -1027,38 +1039,23 @@ func TestOneOfCopy(t *testing.T) {
 	if !in.Equal(out) {
 		t.Fatalf("%#v != %#v", in, out)
 	}
-	if &in.Fields == &out.Fields {
```

Comment:
Non-blocking, since I don't think we use `OneOf` anywhere in our protos yet, but it looks like the tests for the first 6 fields have been removed?